content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -51,6 +51,25 @@ source /etc/profile
 go version
 ```
 ***
+#### Download SODA installer code
```

Comment:
Any reason why we moved this before some prerequisite like python?!
Can we have the pre-req section first?

Reply:
when the python version was updated before ansible installation, the incorrect version of ansible was installed.
This was due to some dependency on the preinstalled version of Python3 when ansible is installed. 
If we install ansible after Python3 is updated an incorrect version of Python is installed.